conf/application.conf

```diff
@@ -66,12 +61,24 @@ mongodb {
   uri = "mongodb://localhost:27017/pega-proof-of-concept-proxy"
 }
 
+pega-proof-of-concept-stubs-uris {
+    submit-payload = "/pega-proof-of-concept-stubs/submit-payload"
+}
+
+pega-proof-of-concept-stubs {
+    baseUrl = "http://localhost:9254"
```

Comment:
we don't need this config value anymore now that you've refactored the code to use the microserivces.services part of the config